orm/src/main/java/org/hibernate/search/query/hibernate/impl/FullTextQueryImpl.java

```diff
@@ -173,7 +181,7 @@ private Loader getProjectionLoader(ObjectLoaderBuilder loaderBuilder) {
 	}
 
 	@Override
-	public ScrollableResults scroll() {
+	public ScrollableResultsImpl scroll() {
```

Comment:
The API requires it; although as noticed in the next commit it seems the SPI `ScrollableResultsImplementor` is good enough now.. changing, thanks!